components/DatasetDetails/DatasetFilesInfo.vue

```diff
@@ -17,9 +30,25 @@ export default {
     datasetDetails: {
       type: Object,
       default: () => {}
-    },
+    }
   },
+
+  data() {
+    return {
+      ctfDatasetNavigationInfoPageId: process.env.ctf_dataset_navigation_info_page_id
+    }
+  }
 }
 </script>
 
-<style lang="scss" scoped></style>
+<style lang="scss" scoped>
+@import '@/assets/_variables.scss';
+.dataset-files-info {
+  .dataset-link {
+    font-size: 12px;
+    font-weight: bold;
+    margin-bottom: 10px;
+    text-decoration: none;
```

Comment:
We're using `em/rem` on SPARC Portal, so I would adjust these values to the following:
```suggestion
    font-size: 0.75rem;
    font-weight: bold;
    margin-bottom: 0.625rem;
    text-decoration: none;
```